DNSSD/Bonjour/include/Poco/DNSSD/Bonjour/Bonjour.h

```diff
@@ -0,0 +1,72 @@
+//
+// Bonjour.h
+//
+// $Id: //poco/1.7/DNSSD/Bonjour/include/Poco/DNSSD/Bonjour/Bonjour.h#1 $
+//
+// Library: DNSSD/Bonjour
+// Package: Implementation
+// Module:  Bonjour
+//
+// Basic definitions for the Poco DNSSD Bonjour library.
+// This file must be the first file included by every other DNSSD Bonjour
+// header file.
+//
+// Copyright (c) 2006-2024, Applied Informatics Software Engineering GmbH.
+// and Contributors.
+//
+// SPDX-License-Identifier:	BSL-1.0
+//
+
+
+#ifndef DNSSD_Bonjour_Bonjour_INCLUDED
+#define DNSSD_Bonjour_Bonjour_INCLUDED
+
+
+#include "Poco/DNSSD/DNSSD.h"
+
+
+//
+// The following block is the standard way of creating macros which make exporting
+// from a DLL simpler. All files within this DLL are compiled with the Bonjour_EXPORTS
+// symbol defined on the command line. This symbol should not be defined on any project
+// that uses this DLL. This way any other project whose source files include this file see
+// POCO_DNSSD_Bonjour_API functions as being imported from a DLL, wheras this DLL sees symbols
+// defined with this macro as being exported.
+//
+#if defined(_WIN32) && defined(POCO_DLL)
+	#if defined(Bonjour_EXPORTS)
+		#define POCO_DNSSD_Bonjour_API __declspec(dllexport)
+	#else
+		#define POCO_DNSSD_Bonjour_API __declspec(dllimport)
+	#endif
+#endif
+
+
+#if !defined(POCO_DNSSD_Bonjour_API)
```

Comment:
Same comment as for Avahi.

OTOH shall the API macro be defined for complete DNSSD and not separately for Avahi and Bonjour?